app/controllers/api/api_controller.rb

```diff
@@ -25,23 +25,23 @@ def dashboard
     end
 
     begin
-      @countries_data = {}
-      countries_calculations = {}
       current_income_list = Income.where(round: round).pluck(:team_name, :amount)
-      previous_income_list = Income.where(round: (round -1)).pluck(:team_name, :amount)
+      previous_income_list = Income.where(round: (round -1)).pluck_to_hash(:team_name, :amount).group_by{|x| x[:team_name]}
 
-      current_income_list.each do |pair|
-        countries_calculations[pair[0]] = pair[1]
-      end
+      countries_data = current_income_list.map do |country, amount|
+        previous_income = previous_income_list[country]
 
-      unless previous_income_list.empty?
-        previous_income_list.each do |pair|
-          countries_calculations[pair[0]] -= pair[1]
+        income_change = if previous_income.empty?
```

Comment:
Derp, realized I can't fix this in #40 since bug is right here.

Anyway, please fix haha.